internal/pkg/deploy/pipeline.go

```diff
@@ -99,9 +101,18 @@ func (b *Build) Init(mfBuild *manifest.Build, mfDirPath string) {
 	if strings.Contains(image, "aarch64") {
 		environmentType = "ARM_CONTAINER"
 	}
+	if mfBuild != nil && !mfBuild.AdditionalPolicy.Document.IsZero() {
+		additionalPolicy, err := yaml.Marshal(&mfBuild.AdditionalPolicy.Document)
```

Comment:
do we need to pass the pointer to the document? could we just do `mfBuild.AdditionalPolicy.Document`?

Reply:
Actually when we `unmarshal` the whole manifest we point to the `Pipeline` variable in which we store the unmarshalled manifest. And while we `Marshal` is back into bytes then also we need pointer to the same variable. So indeed we need `&mfBuild.AdditionalPolicy.Document` else we get empty response from Marshal. 

Does that make sense?